### DISCUSSION TITLE
Error from server (NotFound): secrets "sealos-mongodb-conn-credential" not found

### DISCUSSION THREAD

**Discussion by lizeyou0203** [labring/sealos#4067] [Q&A]
我在执行命令安装sealos-cloud时一直报错: Error from server (NotFound): secrets "sealos-mongodb-conn-credential" not found

命令如下:   sealos run docker.io/labring/sealos-cloud:latest --env cloudDomain="192.168.18.221.nip.io"

报错信息:

2023-10-11T11:59:09 info start to install app in this cluster
2023-10-11T11:59:09 info Executing SyncStatusAndCheck Pipeline in InstallProcessor
2023-10-11T11:59:09 info Executing ConfirmOverrideApps Pipeline in InstallProcessor
2023-10-11T11:59:09 info Executing PreProcess Pipeline in InstallProcessor
2023-10-11T11:59:11 info Executing pipeline MountRootfs in InstallProcessor.
2023-10-11T12:03:08 info Executing pipeline MirrorRegistry in InstallProcessor.
2023-10-11T12:03:09 info Executing UpgradeIfNeed Pipeline in InstallProcessor
namespace/sealos-system created
namespace/sealos created
customresourcedefinition.apiextensions.k8s.io/notifications.notification.sealos.io created
no mongodb uri found, create mongodb and gen mongodb uri
cluster.apps.kubeblocks.io/sealos-mongodb created
serviceaccount/sealos-mongodb-sa created
role.rbac.authorization.k8s.io/sealos-mongodb-role created
rolebinding.rbac.authorization.k8s.io/sealos-mongodb-rolebinding created
waiting for mongodb secret generated
Error from server (NotFound): secrets "sealos-mongodb-conn-credential" not found
Error from server (NotFound): secrets "sealos-mongodb-conn-credential" not found
Error from server (NotFound): secrets "sealos-mongodb-conn-credential" not found
Error from server (NotFound): secrets "sealos-mongodb-conn-credential" not found
Error from server (NotFound): secrets "sealos-mongodb-conn-credential" not found
Error from server (NotFound): secrets "sealos-mongodb-conn-credential" not found
Error from server (NotFound): secrets "sealos-mongodb-conn-credential" not found
Error from server (NotFound): secrets "sealos-mongodb-conn-credential" not found
Error from server (NotFound): secrets "sealos-mongodb-conn-credential" not found
Error from server (NotFound): secrets "sealos-mongodb-conn-credential" not found
Error from server (NotFound): secrets "sealos-mongodb-conn-credential" not found
Error from server (NotFound): secrets "sealos-mongodb-conn-credential" not found
Error from server (NotFound): secrets "sealos-mongodb-conn-credential" not found
Error from server (NotFound): secrets "sealos-mongodb-conn-credential" not found
Error from server (NotFound): secrets "sealos-mongodb-conn-credential" not found

pod列表如下:

NAMESPACE          NAME                                           READY   STATUS    RESTARTS         AGE
calico-apiserver   calico-apiserver-7787d58996-7bc7n              1/1     Running   0                41h
calico-apiserver   calico-apiserver-7787d58996-hwnx8              1/1     Running   0                41h
calico-system      calico-kube-controllers-56674fb757-9j8zw       1/1     Running   5 (11m ago)      42h
calico-system      calico-node-24ttm                              1/1     Running   6 (41h ago)      42h
calico-system      calico-node-4bmqt                              1/1     Running   5 (41h ago)      42h
calico-system      calico-node-5qtfw                              1/1     Running   8 (41h ago)      42h
calico-system      calico-node-5xpxz                              1/1     Running   8 (41h ago)      42h
calico-system      calico-node-mkgqh                              1/1     Running   8 (41h ago)      42h
calico-system      calico-node-pg2g8                              1/1     Running   8 (41h ago)      42h
calico-system      calico-typha-767b6fc74d-4q7pg                  1/1     Running   3 (41h ago)      42h
calico-system      calico-typha-767b6fc74d-96czp                  1/1     Running   0                42h
calico-system      calico-typha-767b6fc74d-ts4dl                  1/1     Running   0                42h
calico-system      csi-node-driver-9b8gs                          2/2     Running   2 (41h ago)      42h
calico-system      csi-node-driver-cz57g                          2/2     Running   0                42h
calico-system      csi-node-driver-g2gnt                          2/2     Running   0                42h
calico-system      csi-node-driver-jcbq8                          2/2     Running   0                42h
calico-system      csi-node-driver-jz6p7                          2/2     Running   2 (41h ago)      42h
calico-system      csi-node-driver-ztrhh                          2/2     Running   0                42h
cert-manager       cert-manager-75f59b674b-26lg2                  1/1     Running   4 (12m ago)      42h
cert-manager       cert-manager-cainjector-5b964db4d8-6dzhd       1/1     Running   4 (13m ago)      42h
cert-manager       cert-manager-webhook-6bdc5556db-chlqh          1/1     Running   0                42h
ingress-nginx      ingress-nginx-controller-9f2p4                 1/1     Running   0                17m
ingress-nginx      ingress-nginx-controller-bm9tx                 1/1     Running   0                17m
ingress-nginx      ingress-nginx-controller-cqgch                 1/1     Running   0                8m44s
ingress-nginx      ingress-nginx-controller-djt7r                 1/1     Running   0                6m39s
ingress-nginx      ingress-nginx-controller-krrm4                 1/1     Running   0                7m27s
ingress-nginx      ingress-nginx-controller-w9kv5                 1/1     Running   0                17m
kb-system          kubeblocks-8d66dc669-5xwzv                     1/1     Running   1 (6m19s ago)    17m
kube-system        coredns-565d847f94-gplsx                       1/1     Running   0                42h
kube-system        coredns-565d847f94-v8jcr                       1/1     Running   0                42h
kube-system        etcd-k8s-master1                               1/1     Running   4                42h
kube-system        etcd-k8s-master2                               1/1     Running   4                42h
kube-system        etcd-k8s-master3                               1/1     Running   0                42h
kube-system        kube-apiserver-k8s-master1                     1/1     Running   33 (13m ago)     42h
kube-system        kube-apiserver-k8s-master2                     1/1     Running   32 (13m ago)     42h
kube-system        kube-apiserver-k8s-master3                     1/1     Running   36 (14m ago)     42h
kube-system        kube-controller-manager-k8s-master1            1/1     Running   55 (6m18s ago)   42h
kube-system        kube-controller-manager-k8s-master2            1/1     Running   65 (10m ago)     42h
kube-system        kube-controller-manager-k8s-master3            1/1     Running   56 (10m ago)     42h
kube-system        kube-proxy-87klh                               1/1     Running   0                42h
kube-system        kube-proxy-ctjmv                               1/1     Running   0                42h
kube-system        kube-proxy-fszss                               1/1     Running   0                42h
kube-system        kube-proxy-jrx8d                               1/1     Running   0                42h
kube-system        kube-proxy-k2s94                               1/1     Running   0                42h
kube-system        kube-proxy-ml422                               1/1     Running   0                42h
kube-system        kube-scheduler-k8s-master1                     1/1     Running   54 (6m7s ago)    42h
kube-system        kube-scheduler-k8s-master2                     1/1     Running   61 (14m ago)     42h
kube-system        kube-scheduler-k8s-master3                     1/1     Running   54 (10m ago)     42h
kube-system        kube-sealos-lvscare-k8s-node1                  1/1     Running   3                42h
kube-system        kube-sealos-lvscare-k8s-node2                  1/1     Running   3                42h
kube-system        kube-sealos-lvscare-k8s-node3                  1/1     Running   3                42h
openebs            openebs-localpv-provisioner-8488c699fb-7qjs9   1/1     Running   10 (9m17s ago)   42h
reflector-system   reflector-7979f4b985-hhr95                     1/1     Running   0                19h
tigera-operator    tigera-operator-547b7f5fc6-bmqsl               1/1     Running   22 (6m15s ago)   42h
zot                zot-55dbc7598b-5n68n                           1/1     Running   0                21m

**Comment by bxy4543** [comment]
这是mongo没起来 用最新的安装脚本试试
